File: python/数组/1608. 特殊数组的特征值.py
from typing import List
class Solution:
    def specialArray(self, nums: List[int]) -> int:
        l = len(nums)
        for i in range(1,l+1):
            count = 0
            for num in nums:
                if num >=i:
                    count+=1
            if count == i:
                return count
        return -1

File: go/array/1608. 特殊数组的特征值.go
package array

func specialArray(nums []int) int {
	for i := 1; i <= len(nums); i++ {
		count := 0
		for _, v := range nums {
			if v >= i {
				count++
			}
		}
		if count == i {
			return count
		}
	}
	return -1
}
